src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoVestingCard/ProgressStepper.tsx

```diff
@@ -0,0 +1,37 @@
+import { Fragment } from 'react'
+import styled from 'styled-components'
+import { Flex } from '@pancakeswap/uikit'
+import { useTranslation } from 'contexts/Localization'
+import { CountdownProps } from '../../types'
+import Step from './Step'
+
+const Spacer = styled.div<{ isPastSpacer?: boolean }>`
+  width: 100%;
+  height: 2px;
+  border-radius: 4px;
+  margin: auto 4px 10px 4px;
+  background-color: ${({ isPastSpacer, theme }) =>
+    isPastSpacer ? theme.colors.textSubtle : theme.colors.textDisabled};
+`
+
+const ProgressStepper: React.FC<CountdownProps> = ({ steps, activeStepIndex }) => {
+  const { t } = useTranslation()
+  return (
+    <Flex>
+      {steps.map((step, index: number) => {
+        const isPastSpacer = index < activeStepIndex
+        const stepText = t(step.text).toUpperCase()
```

Comment:
I don't think this translation would work.

Reply:
Why you think that translation doesn't work? 
Same as Trading Competition component.

https://github.com/pancakeswap/pancake-frontend/blob/2dfb6ffc860738534de3877d7df00d5bfea980e9/src/views/TradingCompetition/components/Countdown/ProgressStepper.tsx#L22-L39